galaxy.yml

```diff
@@ -10,8 +10,12 @@ license_file: LICENSE
 tags:
   - dynatrace
   - oneagent
-  - monitoring
+  - agent
```

Comment:
isn't it to generic?

Reply:
It should be, those are imposed tags, describing the general functionality of our product